webview/src/shared/components/icons/PassFilled.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
Can you add this icon to the Icons story. I'll try to add something to auto generate new icons in there later, but it's still manual for the moment.

Reply:
Added!